R/compat-purrr.R

```diff
@@ -30,6 +30,17 @@ map_cpl <- function(.x, .f, ...) {
   map_mold(.x, .f, complex(1), ...)
 }
 
+walk <- function(.x, .f, ...) {
```

Comment:
I think you can simplify to `map(.x, .f, ...); invisible(.x)`

Reply:
Done.